### ISSUE TITLE
Possible to get in stuck `no active registry` issuance state

### ISSUE THREAD

**Issue by jamshale** [hyperledger/aries-cloudagent-python#3017]
Recently there was a case where issuance stopped working for a traction agent because it was stuck in a state where it was saying it didn't have an active registry, but also wasn't creating one and recovering. The cred def revocation setting was really low at 4, so the registry was rotating often. It didn't fail until many registries had been created.

I was able to recreate this with the demo by setting the revocation size to 4 and rapidly issuing credentials.

**Comment by jamshale** [comment]
This was reported with v1 protocols. I hadn't realized. I'm going to see if I can recreate it with the demo and update the description.

**Comment by jamshale** [comment]
Confirmed this only effects v1 protocols with the demo. Not going to try and fix it right now as v1 is in depreciated status. If it is a priority I should be able to fix it using v2 logic.

**Comment by jamshale** [comment]
I'm going to close this as won't fix, because we don't want to work on v1depreciated protocols. If we decide we should than we can reopen it.

**Comment by jamshale** [comment]
So, I messed up when I was testing/debugging this issue. I had a small code segment that I forgot to discard that was causing the issue with v1 issuance protocol. Both v1 and v2 recover and create the new registry when in the standard flow.

However, I looked at this more and discovered that if a registry rotation happens when unable to connect to the tails server that the agent can also get into this stuck `no registry available` state. This can occur with both v1 and v2 issuance protocols.
